src/engine/test/assets/decoders/fim.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This line is only needed if multiple Yaml documents are defined on the same file, don't hurts though.